assets/components/csi-snapshot-controller/role.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Role
```

Comment:
Leader election is disabled on the csi snapshotter. This being a manual rebase, let's go ahead and drop these (role / binding). I'm looking at the script now to see what's causing them to populate, but they shouldn't be included.